src/main/java/com/artipie/rpm/meta/ChangelogEntry.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Changelog entry.
+ *
+ * @since 0.8.3
+ */
+class ChangelogEntry {
+
+    /**
+     * RegEx pattern of changelog entry string.
+     */
+    private static final Pattern PATTERN = Pattern.compile(
+        "\\* \\w+ \\w+ \\d+ \\d+ (?<author>[^-]+) (?<content>-.*)",
```

Comment:
@olegmoz is there some place where this format is documented? It would be good to add a link to it in the comment above for reference since it is an external constraint for this project

Reply:
@victornoel unfortunately no, I have not found any docs regarding format. But I have found multiple examples in documentation in in this project. So this format is created from examples.